### DISCUSSION TITLE
Write Access

### DISCUSSION THREAD

**Discussion by rpz620** [community/community#69148] [New to GitHub]
hey All - new to Git and we've created a Repository for our third-party dev but he's saying we haven't granted him read/write access even though we have under the Settings, Actions, General workflow permissions. They also are setup as a collaborator. What are we missing?

**Comment by shivangvijay** [comment]
You need to ensure that you've added the developer as a collaborator with the appropriate permissions. Collaborators can be given read, write, or admin access to the repository. To do this, go to the repository settings, then click on Collaborators & teams. Make sure the developer's username or email address is listed there with the correct access level.
Also verify that there are no branch protection rules that could be limiting the developer's access. Go to the repository settings, then "Branches," and review the branch protection rules.

**Comment by rpz620** [comment]
Thank you.  I don’t see any option for access levels.

________________________________
